features/bootstrap/FeatureContext.php

```diff
@@ -17,7 +12,7 @@ public function iAmOnSite($url) {
   public function iSearchFor($searchText) {
     $element = self::$driver->findElement(WebDriverBy::name("q"));
     $element->sendKeys($searchText);
-    $element->submit();
+    $element->sendKeys(WebDriverKeys::ENTER);
```

Comment:
why is this required ?

Reply:
element.submit() on latest Chrome browsers results to the following error: "unknown command: Cannot call non W3C standard command while in W3C mode (UnknownCommandException)"